plugins/module_utils/network/sonic/config/pki/pki.py

```diff
@@ -322,6 +325,20 @@ def _state_merged(self, want, have, diff):
                 }
             )
 
+        # Handle INSTALL for certificates
+        for cert in commands.get("host_cert") or []:
+            if isinstance(cert, dict) and cert.get("file_path") and cert.get("key_path"):
+                requests.append(
+                    {
+                        "path": INSTALL_PATH,
+                        "method": POST,
+                        "data": {"openconfig-pki-rpc:input": 
+                                {"file-path": cert.get('file_path'),
```

Comment:
```suggestion
                                {"file-path": cert.get("file_path"),
```
A little nitpick but for consistency you should use double quotes here